libexec/scoop-update.ps1

```diff
@@ -180,7 +180,7 @@ function update($app, $global, $quiet = $false, $independent, $suggested, $use_c
         $deps | ForEach-Object { install_app $_ $architecture $global $suggested $use_cache $check_hash }
     }
 
-    $version = latest_version $app $bucket $url
+    $version = Get-LatestVersion $app $bucket $url
```

Comment:
Use parameter names as in other uses

Reply:
What 'other uses' means?

Reply:
Get-LatestVersion -App $app -Bucket $bucket - URL $url

- bin\install
- lib\core
- ...

Reply:
Got it.